src/http/wellKnown/internalServerErrorError.ts

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright Inrupt Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+// Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+import { InruptClientError } from "../../clientError";
+import type { ErrorResponse } from "../errorResponse";
+import { ClientHttpError } from "../httpError";
+
+export const INTERNAL_SERVER_ERROR_STATUS = 500 as const;
+
+export type InternalServerErrorErrorResponse = ErrorResponse & {
```

Comment:
The double Error seems a bit odd. Maybe InternalServerErrorResponse?

Reply:
I wasn't sure about this one, even the class name `InternalServerErrorError` looks weird. The error title is `Internal Server Error`, and we get away with it in Java because the base class is `Exception`, not `Error`. All other errors follow the pattern `<Title>Error`, e.g. `NotFoundError`, but I guess here we could make an exception and call it `InternalServerError`, dropping the repeated 'Error' everywhere.

Reply:
I don't have a strong opinion either way. Whatever you think is best.

Reply:
I wasn't really happy with the `ErrorError` repetition, the fact it caught your eye confirmed my impression that it would be best to be more flexible and collapse the name into `InternalServerError`.

Reply:
Fixed in [e91e368](https://github.com/inrupt/solid-client-errors-js/pull/187/commits/e91e368e5ae88bdca3c46ead6a040604f0bf6505)